hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -687,13 +687,21 @@ public void createBucket(
       builder.setDefaultReplicationConfig(defaultReplicationConfig);
     }
 
+    String replicationType = defaultReplicationConfig == null 
+        ? "with server-side default replication type"
```

Comment:
I think this `with` is unnecessary, as the message would look like: `Replication Type set to with server-side default replication type`.

```suggestion
        ? "server-side default replication type"
```

Reply:
This has been fixed in the latest commit

Reply:
Maybe it was fixed locally, but it didn't show up in the PR.  I have applied the suggestion since it's a trivial one.